cmd/virt-handler/virt-handler.go

```diff
@@ -496,6 +504,22 @@ func (app *virtHandlerApp) shouldChangeRateLimiter() {
 	log.Log.V(2).Infof("setting rate limiter to %v QPS and %v Burst", qps, burst)
 }
 
+// Install the SELinux policy when the feature gate that disables it gets removed
+func (app *virtHandlerApp) shouldInstallSELinuxPolicy() {
+	app.semoduleLock.Lock()
```

Comment:
That's a good point, I don't see any callback that would benefit from being called multiple times in parallel...
`semodule` highlighted the issue, as it takes like 20 seconds to install a module, and takes a filesystem-based lock while doing so!
I'd suggest filing an issue to do that as a separate effort, to avoid further delaying this PR.

Reply:
https://github.com/kubevirt/kubevirt/issues/9061